extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -1006,7 +1009,13 @@ export class VideoDocking {
       setImportantStyles(element, {'overflow': 'visible'});
 
       toggle(shadowLayer, true);
-      toggle(overlay, true);
+
+      // TODO(alanorozco): It would be nicer if these didn't get created until
+      // necessary (i.e. not at all if all players on page have their controls
+      // respected).
+      const {respectControls} = devAssert(this.currentlyDocked_).target;
```

Comment:
This might be pretty easy to satisfy in the Bento world. Would be worth discussing how this would translate there.